wix/Build.OpenJDK_generic.cmd

```diff
@@ -307,6 +337,3 @@ SET WIN_SDK_FULL_VERSION=
 SET WIN_SDK_MAJOR_VERSION=
 
 EXIT /b 0
-
-:FAILED
-EXIT /b 2
```

Comment:
Why remove this ? 
There is GOTO FAILED on top of the script

Reply:
?

Reply:
added it back